Drive-by enhancements to docs (#1626)
opensafely-core/ehrql: docs/index.md

*ehrQL* (rhymes with *circle*), the **Electronic Health Records Query Language**,

> is a query language and command line interface that was designed for the specific application domain of EHR data.

Reviewer: Was the original really unclear? Obviously, excessively elliptical constructions can be problematic; but to me that modified sentence is clunkier but no easier to parse.

Author: I've been reading _Developing Quality Technical Information_ and also thinking more about teaching/learning, which includes thinking more about people who don't speak English as their first language. I don't think the original was unclear; I was, however, persuaded that the revision was clearer when I saw a similar example translated into French and German, and how the "that was" made the translations more accurate.

Reviewer: Right yes, this in an interesting point. I wonder if we might end up needing to explicitly distinguish different sorts of writing, with different sorts of goal and audience, within the Bennett Institute. There's definitely a place for maximising non-native readability (of which reference docs would be the obvious example) but I wouldn't  want to eviscerate all personality from our writing more generally.

ehrQL allows researchers to access data sources from primary and secondary care,
as well as from organisations such as the Office for National Statistics (ONS).

> You can find out [why researchers should use ehrQL](https://www.bennett.ox.ac.uk/blog/2023/09/why-ehrql/) on the Bennett Institute blog.

Reviewer 2: As well as being a good idea, it's also written in one of our organisation policy documents that we should always do this.

Author: ```suggestion
You can find out about ehrQL's [philosophy and design goals](https://www.bennett.ox.ac.uk/blog/2023/09/why-ehrql/) on the Bennett Institute blog.
```

Author: My reasoning is that _[Link to answer]_ leads to _Click here_ and _Click here_ is undesirable. However, avoiding _Click here_ is cumbersome, hence the rewrite.

Let's think about the question. What kind of reader would ask, "Why did you decide to create a new language?" What kind of reader would ask, "What are ehrQL's philosophy and design goals?" I'd argue that this section of the docs should be pitched to the second kind of reader, rather than the first.

Author: Suggestion accepted 🙂
